Lint failure on develop

## Description

Lint failure in [this build](https://circleci.com/gh/freedomofpress/securedrop/12574):

```
.python3/.venv/bin/mypy ./securedrop ./admin
securedrop/tests/conftest.py:34: error: Argument 2 to "get" of "Mapping" has incompatible type "None"; expected "str"
```

Due to new release of `mypy` 2 hours ago ;)

--- a/securedrop/tests/conftest.py
+++ b/securedrop/tests/conftest.py
@@ -31,7 +31,7 @@ gnupg_logger = logging.getLogger(gnupg.__name__)
 gnupg_logger.setLevel(logging.ERROR)
 valid_levels = {'INFO': logging.INFO, 'DEBUG': logging.DEBUG}
 gnupg_logger.setLevel(
-   valid_levels.get(os.environ.get('GNUPG_LOG_LEVEL', None), logging.ERROR)
+   valid_levels.get(os.environ.get('GNUPG_LOG_LEVEL', ""), logging.ERROR)
 )
 
 

